src/_js/map.js

```diff
@@ -372,15 +373,19 @@
 		L.ExivaButton.btns = L.exivaButton({
 			crosshairs: this.crosshairs
 		}).addTo(map);
-		L.MarkersButton.btns = L.markersButton({
-			map: _this
-		}).addTo(map);
 		_this._showHoverTile();
-		_this._loadMarkers();
+
+		if (_this.options.markersEnabled === 'true') {
+			L.MarkersButton.btns = L.markersButton({
+				map: _this
+			}).addTo(map);
```

Comment:
Whoops, I reviewed this too quickly :) I think we'll want to show the M button in either case, just not enable the markers by default. So the previous three lines can go outside of the `if`.

Reply:
Done in d1268fc1e528a061a9794d603ac89a56a97ed5fa